functions/daemon.go

```diff
@@ -198,6 +211,12 @@ func messageQueue(ctx context.Context, wg *sync.WaitGroup, server *config.Server
 	err := setupMQTT(server)
 	if err != nil {
 		slog.Error("unable to connect to broker", "server", server.Broker, "error", err)
+
+		// start mqtt fallback goroutine if it is not running
+		if !MQFallbackRunning.Load() {
```

Comment:
if you are using these variable within the same package then you can keep them private